test/test_torch.py

```diff
@@ -3319,6 +3319,30 @@ def rand_tensor(size, dtype, device):
             out = torch.addcmul(a, b, c, value=-1)
             self.assertTrue(not (out.isnan() or out.isinf()))
 
+    def test_addcmul_cpu_scalar_errors(self, device):
+        # Logic is dtype agnostic, so dtype isn't tested
+        alpha = 0.5
+
+        a = torch.rand((2, 2), device=device)
+        b = torch.rand((2, 2), device=device)
+        c = torch.rand((2, 2), device=device)
+        scalar = torch.rand([], device="cpu")
+
+        torch.addcmul(a, b, c, value=alpha)
+        torch.addcmul(a, b, scalar, value=alpha)  # supported on both
```

Comment:
this should already be tested in the test above, yes? If so, then this doesn't have to be here. Or we can move the functionality from the above test here, and change the name of this test to not specify errors.

Reply:
Ok! I'll delete it from the previous function. I also feel like I should just not have called it errors since I test the cpu stuff so I'll change the name too!